### PR TITLE
[IMP] *: code cleanup

### PR DESCRIPTION
**Pull request by Feyensv**
--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/sale_coupon/models/sale_coupon.py [hunk 1]
```diff
@@ -115,8 +115,7 @@ def action_coupon_sent(self):
         }
 
     def action_coupon_cancel(self):
-        for coupon in self:
-            coupon.state = 'cancel'
+        self.state = 'cancel'
```
Comment (smetl) [inline addons/sale_coupon/models/sale_coupon.py]: This diff is not related to this task :p

Comment (Feyensv) [reply addons/sale_coupon/models/sale_coupon.py]: It is a different commit, and it still goes in the direction of the PR title ^^

**Comment by Feyensv** [comment]
@robodoo r+

**Comment by Feyensv** [comment]
@robodoo rebase-ff
